contracts/insurance/Insurance.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: UNLICENSED 
+pragma solidity ^0.7.1; 
+pragma experimental "ABIEncoderV2";
+
+import "../interfaces/ITransferDefinition.sol";
+import "../lib/SafeMath.sol";
+import "../lib/LibChannelCrypto.sol";
+
+/// @title InsurancePayment
+/// @notice This contract allows someone to put up a payment as "insurance"
+///         which can be used to guarantee virtually anything. The insurance
+///         can only be unlocked by a specified mediator, and will be refunded
+///         to the guarantor after a specified expiration period, or if
+///         explicitly cancelled by the receiver.
+
+contract Insurance is ITransferDefinition {
```

Comment:
I know this is a last minute change, but you can simplify this contract by making it extend the `TransferDefinition`, that way you don't have to do the boiler plate `getRegistryInformation` function, and instead can just define the encodings and names. See [here](https://github.com/connext/transfers/blob/6b47ba910e26ffde4324bfc9f1d3dd1048272a5f/contracts/hashlock/Hashlock.sol#L12-L28)

Reply:
Done.